ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml

```diff
@@ -255,3 +255,66 @@ SENSOR_ZTM_ZHM_CO2M_VOCM_PMM:
   - CO2M
   - VOCM
   - PMM
+
+SENSOR_ZTM_ZHM_ZOC:
+  description: "Zone occupant counter with zone temperature & humidity monitoring."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  - ZTM
+  - ZHM
+  - ZOC
+
+SENSOR_ZTM_ZHM_CO2M_VOCM_PML_SPLM:
+  description: "Indoor Air Quality sensor with temperature, humidity, CO2, VOC, particulates (density), illuminance and sound monitoring."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  - ZTM
+  - ZHM
+  - CO2M
+  - VOCM
+  - SPLM
+  uses:
+  - zone_air_aqi_sensor
+  - illuminance_sensor
+  - zone_air_pm2pt5_density_sensor
+  - zone_air_pm10pt0_density_sensor
+
+SENSOR_ZTM_ZHM_CO2M_VOCM_PML:
+  description: "Indoor Air Quality sensor with temperature, humidity, CO2, VOC, particulates (density) monitoring."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  - ZTM
+  - ZHM
+  - CO2M
+  - VOCM
+  uses:
+  - zone_air_pm2pt5_density_sensor
+  - zone_air_pm10pt0_density_sensor
+
+SENSOR_ZTM_ZHM_CO2M_SPLM_PLM_COM_ZSPM_MS_ZAQI_TMPR:
+  description: "Indoor Air Quality sensor with temperature, humidity, CO2, CO, sound, particulates (density), illuminance, formaldehyde, pressure sensors with air quality, vape, TVOC indexes and motion & tamper detection."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  - ZTM
+  - ZHM
+  - CO2M
+  - SPLM
+  - COM
+  - ZSPM
+  uses:
+  - illuminance_sensor
+  - zone_air_pm2pt5_density_sensor
+  - zone_air_formaldehyde_concentration_sensor
```

Comment:
Add this field as opt_uses on the existing `VOCM` type (and then implement that type here, making fields MISSING as needed)